Adição dos scripts e módulos

sa0tome opened the pull request:
Acho que falta criar um diretório pros meus scripts, mas não consigo fazer isso (não há opção).

sa0tome committed:
Adição dos scripts e módulos

thiagogomesverissimo reviewed (commented):
muito bom. 
Não rodei ainda, mas parece ok.

sa0tome committed:
Atualização de scripts com +2 APIs 

Atualização do main script normalizateses.py com mais dois módulos: addnid e tesesarea. O primeiro insere a coluna nid e popula com os valores respectivos para cada pessoa, e o segundo módulo normaliza a coluna "área" de acordo com os cursos da pós-graduação.

sa0tome committed:
Atualização do arquivo teses.csv 

Normalização da coluna "área".

sa0tome commented:
Adicionei alguns scripts para normalização da área e adição da coluna NID, assim como uma atualização do arquivo teses.csv com a coluna área normalizada.
